### PR TITLE
I18n en translations reference

### PR DESCRIPTION
**Pull request by carygravel**
This is still a work in progress. Just wanted to be transparent about how far I had got. Note that I started with @duckfullstop 's branch. If his commits are a problem, I can rebase/remove them.

### CODE REVIEW

**Comment by duckfullstop** [comment]
Could you rebase onto main without my commits? Should keep things a little tidier, especially as you're touching a section of the documentation that I haven't.

**Comment by carygravel** [comment]
That's reference finished, I think. Please take a look!

**Review by duckfullstop — CHANGES_REQUESTED** [review]
Pending the given changes this'll be a LGTM.

File: i18n/en/docusaurus-plugin-content-docs/current/reference/configuration/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+generated power is interesting, but has no effect on the function, with the
+this exception:
+https://docs.evcc.io/docs/guides/meters#ich-habe-eine-pv-anlage-aber-keinen-netzanschluss-z%C3%A4hler-kann-ich-evcc-trotzdem-nutzen
+
+The surplus power is compared with the minimum power required to charge. If
+this is sufficient, the charging process is started.
+
+The minimum power required to charge is calculated from the values `minCurrent`
+and `phases`, defined per `loadpoint` (a group of colocated chargers)
+See https://docs.evcc.io/docs/reference/configuration/loadpoints for more
```
[the remark this answers is not in the document]
Comment (naltatis) [reply i18n/en/docusaurus-plugin-content-docs/current/reference/configuration/index.md:80]: Yes, this should definitly be a relative link like this: 

```suggestion
See [`loadpoints`](./loadpoints)
```

**Comment by naltatis** [comment]
Looks very nice! Do you have anything beside the linking comments open? Otherwise I would merge this.

**Review by naltatis — APPROVED** [review]

**Comment by duckfullstop** [comment]
@carygravel Any idea what state this is in at the moment? Could just merge this and then tackle the links in a separate issue

**Comment by naltatis** [comment]
@carygravel As suggested by @duckfullstop, I'll merge this now and we can update in smaller increments from there.

**Pull request merged**